common/sketch/src/main/java/org/apache/spark/util/sketch/BloomFilter.java

```diff
@@ -39,6 +43,28 @@
  * The implementation is largely based on the {@code BloomFilter} class from guava.
  */
 public abstract class BloomFilter {
+
+  public enum Version {
+    /**
+     * {@code BloomFilter} binary format version 1 (all values written in big-endian order):
+     * - Version number, always 1 (32 bit)
+     * - Total number of words of the underlying bit array (32 bit)
+     * - The words/longs (numWords * 64 bit)
+     * - Number of hash functions (32 bit)
```

Comment:
Nit: Scaladoc requires an extra space before `-` to form an unordered list. I'll fix this one in #10911.